frontend/src/components/auth/Login.tsx

```diff
@@ -24,55 +48,66 @@ const Login = (): React.ReactElement => {
     setAuthenticatedUser(user);
   };
 
-  const onSignUpClick = () => {
-    history.push(SIGNUP_PAGE);
-  };
-
   if (authenticatedUser) {
     return <Redirect to={HOME_PAGE} />;
   }
 
-  return (
-    <div style={{ textAlign: "center" }}>
-      <h1>Login</h1>
-      <form>
-        <div>
-          <input
-            type="email"
-            value={email}
-            onChange={(event) => setEmail(event.target.value)}
-            placeholder="username@domain.com"
-          />
-        </div>
-        <div>
-          <input
-            type="password"
-            value={password}
-            onChange={(event) => setPassword(event.target.value)}
-            placeholder="password"
-          />
-        </div>
-        <div>
-          <button
-            className="btn btn-primary"
-            type="button"
-            onClick={onLogInClick}
+  const title = isAdmin ? "Admin Login" : "Teacher Login";
+  const image = isAdmin ? ADMIN_SIGNUP_IMAGE : TEACHER_SIGNUP_IMAGE;
+  const form = (
+    <VStack width="50%" gap={4}>
+      <Text textStyle="subtitle2" textAlign="center" pb={4}>
+        Enter your login credentials below to continue
+      </Text>
+      {loginError && <FormError message="Please ensure fields are filled" />}
+
+      <FormControl isRequired isInvalid={loginError && !email}>
+        <FormLabel color="grey.400">Email Address</FormLabel>
+        <Input
+          type="email"
+          value={email}
+          onChange={(e) => setEmail(e.target.value)}
+          placeholder="Enter Email Address"
+        />
+      </FormControl>
+
+      <FormControl isRequired isInvalid={loginError && !password}>
+        <FormLabel color="grey.400">Password</FormLabel>
+        <Input
+          type="password"
+          value={password}
+          onChange={(e) => setPassword(e.target.value)}
+          placeholder="Enter Password"
+        />
+      </FormControl>
+
+      {/* TODO: add correct routing */}
+      <Flex align="left">
+        <RouterLink textAlign="left" color="blue.300" to="/">
+          Forgot Password?
+        </RouterLink>
+      </Flex>
```

Comment:
I just left this in the center since we are omitting the `Remember Me` feature! We can get design to look at it and change it if they want.